src/bin/urdf-viz.rs

```diff
@@ -355,10 +355,14 @@ impl UrdfViewerApp {
                 }
             }
             Key::Z => {
+                let mut origin = self.robot.origin();
+                origin.translation = na::Translation3::new(0.0, 0.0, 0.0);
+                origin.rotation = na::UnitQuaternion::from_euler_angles(0.0, 0.0, 0.0);
+                self.robot.set_origin(origin);
```

Comment:
I think you can do like below (I've not checked anything).

```suggestion
              self.robot.set_origin(na::Isometry::identity());
```

Reply:
Thanks. That's works well.